SofaKernel/framework/sofa/core/CMakeLists.txt

```diff
@@ -130,6 +130,13 @@ set(HEADER_FILES
     topology/TopologyChange.h
     topology/TopologyElementHandler.h
     topology/TopologyHandler.h
+    topology/MapTopology.h
+    topology/CMTopologyChange.h
```

Comment:
@untereiner do you think it is possible to have the topology/CM* files put in a plugin so we can do stuff like that:
```xml
<RequiredPlugin name="CGognTopology">
<RequiredPlugin name="CGognForceField">
```

Where the plugin CGognForceField depend on CGOgnTopology ?